srcpkgs/python3-libtmux/template

```diff
@@ -11,7 +11,7 @@ maintainer="Alexander Egorenkov <egorenar-dev@posteo.net>"
 license="MIT"
 homepage="https://github.com/tmux-python/libtmux"
 distfiles="${PYPI_SITE}/l/libtmux/libtmux-${version}.tar.gz"
-checksum=d82cf391097eb69d784d889d482bb99284b984aa6225276a3dc1af8c1460dd3c
+checksum=cf352a645ebeaa5046ad0593cc01d691d77f8b3af1660327a6433cc6276ab796
 # Disable check which uses pip to pull external python packages
```

Comment:
this is probably caused by not specifying checkdepends properly. maybe it can be fixed?

Reply:
`checkdepends="python3-pytest-mock tmux"`
...
...
`make_check_pre="env PYTEST_PLUGINS=libtmux.pytest_plugin"`

Reply:
can you add this ^ change too?